packages/@cdktf/provider-generator/lib/get/generator/emitter/struct-emitter.ts

```diff
@@ -19,6 +20,21 @@ export class StructEmitter {
     }
   }
 
+  // Due to https://github.com/hashicorp/terraform-plugin-sdk/commit/2387eb85e32c064b4a62718c9f5c80bf00dc7fb9 all
+  // resources from providers using the old SDK have the id field by default
+  // We have no way to distinguish them through the provider schema, so a word of warning for our users
+  private warnAboutIdField(att: AttributeModel) {
+    if (att.name === "id") {
+      this.code.line(`*`);
+      this.code.line(
+        `* Please be aware that the id field is automatically added to all resources in providers using a SDK version below 2.`
```

Comment:
```suggestion
        `* Please be aware that the id field is automatically added to all resources in Terraform providers using a Terraform provider SDK version below 2.`
```

Reply:
Make it a bit more clear, so users don't wonder which SDK version exactly (if they are not familiar with what happens under the hood)